app/views/completed_transaction/_assisted_digital_satisfaction_survey.html.erb

```diff
@@ -1,163 +1,130 @@
-      Neither satisfied or dissatisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-dissatisfied" value="2">
-      Dissatisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-very-dissatisfied" value="1">
-      Very dissatisfied
-    </label>
-  </fieldset>
+  <% other_input = capture do %>
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: "Tell us who the other person was"
+      },
+      name: "service_feedback[assistance_provided_by_other]",
+      id: "other-specified"
+    } %>
 
-  <br />
+    <%= other_person %>
+  <% end %>
 
-  <fieldset>
-    <legend><h2>Do you have any ideas for how this service could be improved?</h2></legend>
-    <br />
-    <label for="improvement-comments" class="visuallyhidden">Your comments</label>
-    <textarea name="service_feedback[improvement_comments]" class="full-size counted" id="improvement-comments"></textarea>
-    <p id="improvement-commentscounter" class="hint" aria-live="polite" aria-atomic="false">(Limit is 1200 characters)</p>
-  </fieldset>
+  <% did_you_receive_assistance = capture do %>
+    <%= render "govuk_publishing_components/components/textarea", {
+      label: {
+        text: "What assistance did you receive?",
+        bold: true
+      },
+      name: "service_feedback[assistance_received_comments]",
+      id: "assistance-received-comments"
+    } do %>
+      <span class="govuk-hint">Limit is 1200 characters</span>
+    <% end %>
+    <%= render "govuk_publishing_components/components/radio", {
+      name: "service_feedback[assistance_provided_by]",
+      heading: "Who provided the assistance?",
+      heading_size: "s",
+      items: [
+        {
+          value: "friend-relative",
+          text: "A friend or relative"
+        },
+        {
+          value: "work-colleague",
+          text: "A work colleague"
+        },
+        {
+          value: "government-staff",
+          text: "A staff member of the responsible government department",
+          conditional: government_staff
+        },
+        {
+          value: "other",
+          text: "Other (please specify)",
+          conditional: other_input
+        },
+      ]
+    } %>
+  <% end %>
 
-  <br />
+  <%= render "govuk_publishing_components/components/radio", {
+    name: "service_feedback[assistance_received]",
+    heading: "Did you receive any assistance to use this service today?",
+    heading_size: "s",
+    items: [
+      {
+        value: "yes",
+        text: "Yes",
+        conditional: did_you_receive_assistance
+      },
+      {
+        value: "no",
+        text: "No"
+      }
+    ]
+  } %>
+
+  <%= render "govuk_publishing_components/components/radio", {
```

Comment:
Is this worth using the `assistance_received` partial since the answers are the same?

Reply:
If I understand correctly, you're suggesting repeating the use of the partial that's already used further up? I could modify the partial to accept values for the question text and name attributes for each component (they differ slightly) but the partial was created for a very specific problem, so I'm not sure. Is that what you mean?

Reply:
That's what I mean - give the partial the  values for `name` and for the heading text so you can use the partial for each set of satisfied -> unsatisfied radio buttons.